site/docs/patterns/preferences-dialog.mdx

```diff
@@ -0,0 +1,178 @@
+      { href: "https://www.figma.com/file/N6cSYycc4z6XkCjVvc1kIA/Preferences-Dialog-Pattern?type=design&node-id=157-40560&mode=design&t=eq3OapLjaquXm3UP-0", label: "Preferences Dialog stickersheet", internal: true },
+    ]
+---
+
+A preferences dialog allows you to display, group and navigate application preferences within a responsive modal dialog window. This pattern introduces guidelines to make the preferences dialog consistent and usable.
+  
+  <Diagram
+    src="/img/patterns/preferences-dialog/preferences-dialog.png"
+    alt="Preferences Dialog example"
+    border
+  />
+
+## When to use
+
+Any preference settings in your platform or application should ideally be found within a single centralised preferences dialog. These settings could be global, application-wide settings or settings local to a specific window or widget (such as a grid or comments feed). If accessing from a specific window, the relevant settings should immediately be displayed when the preferences dialog opens.
+
+Furthermore, we recommend implementing consistent behavior around applying preferences, whether applying changes immediately or requiring the user to confirm any changes using the button bar actions. 
+
+## How to build
+
+### Anatomy
+
+A preferences dialog comprises four key features: 
+
+1. [Dialog](../components/dialog): a window with a content area that opens over the application content, focusing the user’s attention on a particular task or piece of information
+2. [Navigation item](../components/navigation-item): a link that allows users to navigate to distinct panel of content
+3. [Button bar](./button-bar): displays possible actions for users to take when they reach the end of a task. Use the [button bar pattern](./button-bar) to understand how to implement actionable interactions within dialogs across your applications
+4. [Parent-child layout](../components/parent-child-layout): positions the navigation and content within a preference dialog
+
+  <Diagram
+    src="/img/patterns/preferences-dialog/anatomy.png"
+    alt="Preferences Dialog navigation and content."
+    border
+  />
+
+### Layout construction
+
+<ImageSwitcher
+  images={[
+    {
+      src: "/img/patterns/preferences-dialog/layout.png",
+      alt: "Example showing Preferences Dialog's layout construction.",
+    },
+    {
+      src: "/img/patterns/preferences-dialog/layout-grid.png",
+      alt: "Example showing Preferences Dialog's layout construction with grid.",
+    },
+  ]}
+  label="Layout grid"
+/>
+
+<ExampleContainer type="positive" customPillText="Best practices">
+  In the [Parent-child layout](../components/parent-child-layout), the parent container (containing the navigation) typically remains fixed in width when the user resizes the browser. You can apply an independent layout grid to align the content with the child container. This will enable it to adapt responsively based on the viewport size.
```

Comment:
```suggestion
  In the [parent-child layout](../components/parent-child-layout), the parent container (containing the navigation) typically remains fixed in width when the user resizes the browser. You can apply an independent layout grid to align the content with the child container. This will enable it to adapt responsively based on the viewport size.
```